tests/scaffoldings/dotd/docker-compose.yaml

```diff
@@ -10,9 +10,8 @@ services:
     environment:
       PGUSER: another_odoo
       PGPASSWORD: anotherodoopassword
-      PGHOST: postgresql
     depends_on:
-      - postgresql
+      - postgresql:manual_db_host
```

Comment:
BTW [This is not a valid syntax](https://travis-ci.org/Tecnativa/docker-odoo-base/builds/262422829#L4924-L4934), although you have to revert this anyway (see above)